test/fixtures/bulletChartDataBuilder.js

```diff
@@ -35,6 +35,50 @@ define(function(require) {
             return new this.Klass(attributes);
         };
 
+        this.withManyCPUData = function () {
```

Comment:
for consistency sake, let's have a json file for this dataset as well

Reply:
I am actually thinking of dumping the JSON files.

They add a level of indirection that is a pain, especially when dealing with tests and examples. It is true that there are a couple of datasets (line and area chart) that are large, but most of the time is not a big deal and our IDE's collapsing features help a lot.

what do you think?

Reply:
Sounds good to me. If we were using TypeScript dataset types could be highlighted if we were to place a wrong key or value type, so I would go with not using json files as well.

Reply:
Cool, I will go ahead and refactor that on the RC